tests/recommendations_endpoint_test.py

```diff
@@ -0,0 +1,142 @@
+from base_app.base import app
+from dotenv import load_dotenv
+load_dotenv()
+
+def test_user_selects_multiple_preferences_returns_successful_response():
+    lat = '39.7392358'
+    long = '-104.990251'
+    cuisine = 'indian'
+    price = '2'
+    url = '/api/v1/recommendations?latitude={}&longitude={}&categories={}&price={}'.format(lat, long, cuisine, price)
+
+    client = app.test_client()
+    response = client.get(url)
+
+    assert response.status_code == 200
+
+    response_body = response.json
+    price = len(response_body['data']['attributes']['price'])
```

Comment:
Is price returned as a string of symbols and that's why length is being used?

Reply:
That's exactly why